docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'HowTo: Manage merchant custom prices for products via data import'
+last_updated: Feb 04, 2022
+template: howto-guide-template
+---
+
+{% info_block warningBox "B2B only" %}
+
+You can apply the approaches described in this article only if your project follows the B2B model, as the [merchant custom prices](https://docs.spryker.com/docs/scos/user/features/{{site.version}}/merchant-custom-prices-feature-overview.html) is only the B2B feature. Also, keep in mind that these approaches don't work for the Marketplace product offers.
+
+{% endinfo_block %}
+
+Let's assume you have a [default price](https://docs.spryker.com/docs/scos/user/features/202108.0/scheduled-prices-feature-overview.html#price-types) for products, and you have merchants for whom you want to make a certain percentage discount from the default price.
```

Comment:
Should we not put `{{site.version}}` for the `../202108.0/..` in the URL? If yes, this must be corrected in all URL below